### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Default Documents_Keyboard: Focus order is not proper after pressing Enter on Edit button

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#112]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**MAS ID Violated:** MAS2.4.3, MAS1.3.2

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Default Documents and press enter
6) Validate that focus order is as per flow of UI design in meaningful and logical sequence.

**Actual Result:**
Focus order is not proper after pressing Enter on Edit button. 
After pressing Enter on Edit button and press Tab since user wants to edit the File Name so focus should go to the text box in order to edit the file name but here after Edit Button focus is going on the OK button which is incorrect

**Expected Result:**
Focus order should be as per flow of UI design in meaningful and logical sequence.

****User Impact**:** 
Visually Impaired Users will face difficulty in navigating through the contents and controls if the focus order is not correct 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachments:**
[Bug_112_Focus Order is not proper after pressing Enter on Edit button .pptx](https://github.com/Microsoft/IIS.WebManager/files/2326664/Bug_112_Focus.Order.is.not.proper.after.pressing.Enter.on.Edit.button.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.2; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #Edge; #Narrator; #MAS2.4.3; #Keyboard

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
